telegram/ext/handler.py

```diff
@@ -143,14 +144,19 @@ def handle_update(
                 the dispatcher.
 
         """
+        should_run_async = self.run_async
```

Comment:
I think we can rename `should_run_async` to `run_async`. the `should` doesn't give any extra information imho

Reply:
Thanks for reviewing.. I'll address requested changes soon..